tests/unit/connector/test_opensearch_output.py

```diff
@@ -360,3 +366,27 @@ def test_message_backlog_is_cleared_after_it_was_written(self):
         self.object._config.message_backlog_size = 1
         self.object.store({"event": "test_event"})
         assert len(self.object._message_backlog) == 0
+
+    @pytest.mark.skip(reason="This test is only for local debugging")
```

Comment:
Can't we turn this into a similar acceptance test like you did for Kafka? So that the tests starts and opensearch instance against which this test can actually run then.

Reply:
the overhead is pretty high for this... opensearch starts slow.... I would prefer to not do this